common/djangoapps/course_modes/helpers.py

```diff
@@ -13,6 +15,43 @@
 DISPLAY_AUDIT = "audit"
 DISPLAY_PROFESSIONAL = "professional"
 
+COURSE_INFO_PAGE = "COURSE_INFO"
+SUCCESS_ENROLL_PAGE = "ENROLL_SUCCESS"
+
+
+def get_mktg_url(page):
+    """
+    Returns course marketing full url
+    """
+    lang = ''
+
+    if get_language() != 'ar':
+        lang = 'en/'
+
+    url = '{root}{lang}{page}'.format(
+        root=settings.MKTG_URLS['ROOT'],
+        lang=lang,
+        page=page
+    )
+    return url
+
+
+def get_mktg_for_course(page, course_id):
```

Comment:
@OmarIthawi I think this is supposed to do the job? https://github.com/edx/course-discovery/pull/75/files